SimplaeWAWS.Models/ApiApp.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Newtonsoft.Json;
```

Comment:
Hmm IIRC, there was some cyclic dependency cropping up. 
Since the original project hasnt been removed. Will move these changes back to a renamed `SimpleWAWS.Models` project and investigate how to remove the dependency. Alternately will remove the `SimpleWAWS.Models` project